sphinxcontrib/plantuml.py

```diff
@@ -613,15 +615,30 @@ def _convert_eps_to_pdf(self, refname, fname):
         raise PlantUmlError('error while running epstopdf\n\n%s' % serr)
     return refname[:-4] + '.pdf', fname[:-4] + '.pdf'
 
+def _svg_to_pdf_default(node, refname, fname):
+    raise PlantUmlError('plantuml_latex_output_format svg was selected but no ImageConverter for svg to pdf was found.')
+
+def _svg_to_pdf_init(app):
+    for transform in app.registry.get_post_transforms():
+        if issubclass(transform, ImageConverter):
+            for source_type, target_type in transform.conversion_rules:
+                if 'svg' in source_type and 'pdf' in target_type:
+
+                    def _svg_to_pdf(node, refname, fname, transform=transform):
+                        transform(node.document).convert(fname, fname[:-4] + '.pdf')
+                        return (refname[:-4] + '.pdf', fname[:-4] + '.pdf')
+
+                    _KNOWN_LATEX_FORMATS['svg'] = ('svg', _svg_to_pdf)
+                    return
 
 _KNOWN_LATEX_FORMATS = {
     'eps': ('eps', lambda self, refname, fname: (refname, fname)),
     'pdf': ('eps', _convert_eps_to_pdf),
     'png': ('png', lambda self, refname, fname: (refname, fname)),
     'tikz': ('latex', lambda self, refname, fname: (refname, fname)),
+    'svg': ('svg', _svg_to_pdf_default)
```

Comment:
Perhaps, the `pdf` rule can be overridden if `ImageConverter` found? It seems a bit odd that the `svg` format generates a pdf. If needed, maybe we can add `plantuml_use_image_converter_to_generate_pdf` config or something.